### ISSUE TITLE
Fatal server crash with Energy Inhibitors

### ISSUE THREAD

**Issue by Aryx3D** [MeridiusIX/Modular-Encounters-Systems#242]
Activation of an energy inhibitor via MES action causes a server crash.
Exception:
```
12:37:58.1703 [FATAL]  Initializer: System.NullReferenceException: Object reference not set to an instance of an object.
   at ModularEncountersSystems.Entities.PlayerEntity.IsPlayerStandingCharacter()
   at ModularEncountersSystems.BlockLogic.EnergyInhibitor.RunTick100()
   at ModularEncountersSystems.BlockLogic.BaseBlockLogic.Run()
```

**Comment by AutoTeq** [comment]
We have also seen this issue as well.
[message.txt](https://github.com/MeridiusIX/Modular-Encounters-Systems/files/9255515/message.txt)

**Comment by MeridiusIX** [comment]
This was addressed in 2.66.5 - please make sure your server version of the mod is upto date
